app/src/App/routes/index.tsx

```diff
@@ -1174,6 +1175,20 @@ const perWorkPlanForm = customWrapRoute({
     },
 });
 
+const aboutRedirect = customWrapRoute({
+    parent: rootLayout,
+    path: '/about',
+    component: {
+        render: () => import('#views/Resources'),
+        props: {},
+    },
+    wrapperComponent: Auth,
+    context: {
+        title: 'Resources',
+        visibility: 'anything',
+    },
+});
+
```

Comment:
Let's not render Resources page directly, we should use `Navigate` component to route to the resources page.
For example, https://github.com/IFRCGo/go-web-app/blob/6d755498fa096679ebd817fdb1efe9d7e94a2398/app/src/App/routes/index.tsx#L1178C1-L1193C4